pkg/sources/github/github.go

```diff
@@ -1492,8 +1492,10 @@ func (s *Source) scanTarget(ctx context.Context, target sources.ChunkingTarget,
 		SourceMetadata: &source_metadatapb.MetaData{
 			Data: &source_metadatapb.MetaData_Github{Github: meta},
 		},
-		Verify: s.verify}
-	return handlers.HandleFile(ctx, readCloser, &chunkSkel, reporter)
+		Verify: s.verify,
+	}
+	fileCtx := context.WithValues(ctx, "path", meta.GetFile())
```

Comment:
I suspect this method is only called via the Enterprise flow. Source-specific changes would presumably need to be made for each source.